gax-grpc/src/main/java/com/google/api/gax/grpc/ApiException.java

```diff
@@ -27,53 +27,31 @@
  * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
-package com.google.api.gax.grpc;
+package com.google.api.gax.rpc;
 
 import com.google.api.core.BetaApi;
-import com.google.common.base.Preconditions;
-import io.grpc.Status;
 
-/**
- * Represents an exception thrown during an RPC call.
- *
- * <p>It stores information useful for functionalities in {@link UnaryCallable} and {@link
- * StreamingCallable}. For more information about the status codes returned by the underlying grpc
- * exception see
- * https://github.com/grpc/grpc-java/blob/master/core/src/main/java/io/grpc/Status.java
- */
+/** Represents an exception thrown during an RPC call. */
 @BetaApi
 public class ApiException extends RuntimeException {
   private static final long serialVersionUID = -725668425459379694L;
```

Comment:
I think that deleting a field means the serialVersionUID needs to be updated: https://docs.oracle.com/javase/7/docs/platform/serialization/spec/version.html#5172